pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -0,0 +1,73 @@
+{
+boost,
+cmake,
+cm256cc,
+fetchFromGitHub,
+fftwFloat,
+glew,
+libav,
+libiio,
+libopus,
+libpulseaudio,
+libusb,
+limesuite,
+mkDerivation,
+ocl-icd,
+opencv3,
+pkgconfig,
+pkgs,
+qtbase,
+qtmultimedia,
+qtwebsockets,
+serialdv,
+stdenv,
+wrapQtAppsHook,
+}:
+
+let
+
+  codec2 = pkgs.codec2.overrideAttrs (old: {
```

Comment:
```suggestion
  codec2' = codec2.overrideAttrs (old: {
```

Reply:
You can't do that or it does an infinite recursion and won't build ^

Reply:
You also have to replace `codec2` with `codec2'` in the `buildInputs`, then it will work.

Reply:
Oh ok, i wasn't wearing my glasses lol
after using rg it seems like it is the standard way to overrid pkgs and not use pkgs